core/base/ripsPersistenceDiagram/RipsPersistenceDiagram.h

```diff
@@ -0,0 +1,57 @@
+/// \ingroup base
+/// \class ttk::RipsPersistenceDiagram
+/// \author Mattéo Clémot <matteo.clemot@univ-lyon1.fr>
+/// \date January 2024.
+///
+/// \brief TTK VTK-filter that computes the persistence diagram of a Rips
+/// complex.
+///
+/// \param Input Input table (vtkTable)
+/// \param Output PersistenceDiagram (vtkUnstructuredGrid)
+///
+/// This module defines the %RipsPersistenceDiagram class that takes a point
+/// cloud or a distance matrix (vtkTable) and computes the persistence diagram
```

Comment:
remove the mention of `vtkTable` (in principle, nothing assumes the usage of VTK in the base classes)